tests/foreman/ui/test_dashboard.py

```diff
@@ -259,3 +259,35 @@ def test_positive_user_access_with_host_filter(
         assert len(errata_values) == 1
         assert errata_values[0]['Type'] == 'security'
         assert settings.repos.yum_6.errata[2] in errata_values[0]['Errata']
+
+
+@pytest.mark.tier2
+@pytest.mark.upgrade
```

Comment:
@lpramuk Is the upgrade mark desired here?

Reply:
@ogajduse why do you ask me ? I'm not the one who's adding the test

I can only speculate that this test plays important role in a specific component coverage ....

Reply:
Thanks, @lpramuk. I did not know the process around the upgrade markers.

@adarshdubey-star I am in favor of removing the marker from this test. Is there any reason to keep it?

Reply:
I can't see the need for that marker either.